Xamarin.Forms.Platform.SkiaSharp/Native/ListView.cs

```diff
@@ -7,6 +7,12 @@ public class ListView : SKView
 
 		public override SKSize Measure(SKSize available)
 		{
+			var bounds = new SKRect()
+			{
+				Size = new SKSize(available.Width, 50)
```

Comment:
Maybe it's a good idea to move this to a constant and that's easier to change (50 for minimum Height).